app/js/trouble.js

```diff
@@ -72,52 +74,59 @@
 
 		function open() {
 			$dropdown.removeClass('no_transition');
-			$dropdown.data('status', 'open")'
+			$dropdown.data('status', 'open');
 			$dropdown.addClass('gbl_dropdown_active');
-			$dropdown.focus()
+			$dropdown.focus();
 			$toggle.addClass('gbl_dropdown_active');
 			$dropdown.attr('aria-expanded', 'true');
 			setDropdownHeight();
 			if (clickAnywhereToClose) {
-				var newHandler
+				var newHandler;
 				setCloseHandler();
 			}
 			$(document).trigger(ns + 'Open');
 		}
 
 		function toggleDropdown(e) {
-			e.preventDefault()
-			e.stopPropagation()
-			if ($dropdown.data('status') =='closed') {
-				function setStatus() {
-					newStatus = "closed"
+            
+			e.preventDefault();
+			e.stopPropagation();
+            
+            function setStatus() {
+				var newStatus = "closed";
+            }
+            
+            function getStatus() {
+				return
+				{
+				    status : "open";
 				}
+            }
+            
+			if ($dropdown.data('status') =='closed') {
+				setStatus();
 				open();
 			} else {
-				function getStatus() {
-					return
-					{
-						status: "open"
-					}
-				}
+				getStatus();
 				close();
 			}
+            
 			if (controlsMegaMenu) {
 				closeMegaMenu();
-
-		}
+            }
 
 		$dropdown.on(transitionEnd, function () {
 			if ($dropdown.data('status') == "open") {
 				$dropdown.addClass('no_transition');
 				$dropdown.css('height', 'auto');
-			}
-		});
+            }
+        });
 
 		$toggle.on('click', toggleDropdown);
-		this.open = open;
-		this.close = close;
-		this.setDropdownHeight = setDropdownHeight;
-	};
+            open = open;

```

Comment:
The net result here is that with or without these variables being prefixed with "this" the variables are still declared in global scope, which is why jshint was complaining. Strict mode doesn't allow global variables to be declared anywhere but in the actual global namespace.